apps/web/pages/api/basenames/frame/assets/retrySearchFrameImage.png.tsx

```diff
@@ -0,0 +1,118 @@
+import { ImageResponse } from '@vercel/og';
+import { NextRequest } from 'next/server';
+import { isDevelopment } from 'apps/web/src/constants';
+import { openGraphImageHeight, openGraphImageWidth } from 'apps/web/src/utils/opengraphs';
+import retrySearchImageBackground from 'apps/web/public/images/frames/basenames/retry-search-image.png';
```

Comment:
same

Reply:
same, got this working in a closer directory